src/[sandbox]/index.html

```diff
@@ -5,6 +5,16 @@
   <meta charset="utf-8">
   <title>CJaas Common Components Playground</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!-- dependencies for walkin component -->
+  <script
+    type="module"
+    src="https://cj.exp.bz/event-listener/ref-cjaas-requests.js"
```

Comment:
do we need some sort of README for this component since it requires some additional setup?

Reply:
Good idea, I can add one.